src/Ganeti/HTools/Loader.hs

```diff
@@ -372,6 +373,14 @@ clearDynU cdata@(ClusterData _ _ il _ _) =
   let il2 = Container.map (\ inst -> inst {Instance.util = zeroUtil }) il
   in Ok cdata { cdInstances = il2 }
 
+-- | Update cluster data to use static node memory on KVM.
+setStaticKvmNodeMem :: ClusterData -> Int -> Result ClusterData
+setStaticKvmNodeMem cdata@(ClusterData _ nl _ _ _) staticNodeMem =
```

Comment:
Yeah, after seeing the rest of the code, nl is used plenty (and still can be confusing) so I'm okay with keeping it. However, in *most* (not all, unfortunately) of our codebase I think we tend to avoid using ' as that is ambiguously defined as either a strictness identifier or as you said a mutated version of a variable. I've seen "new_node" or "n1" "n2" being used in other parts of the code, maybe that's better.